test/README.md

```diff
@@ -16,18 +16,6 @@ make
 # or
```

Comment:
Can we have an Overview section where we explain what is the test manager and how it is used for integration tests ?
It will help people understand this module's purpose.